src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,31 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    completed,
+    title,
+    user,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': completed },
+      )}
```

Comment:
Remove unnecessary linebreaks
```suggestion
      className={classNames('TodoInfo',{ 
       'TodoInfo--completed': completed 
       })}
```